### DISCUSSION TITLE
Authentification error

### DISCUSSION THREAD

**Discussion by jmfayard** [gavvvr/obsidian-imgur-plugin#70] [Getting help]
Hello, the imgur plugin has worked flawlessly for months, I have used it a lot

Nowdays nothing works and I don't know on which side things got broken. 
Imgur ? The plugin ? I have uploaded too much ?

I can't upload a thing

I can't go in the settings and authenticate again.
The web page fails with the error

> Error 503 first byte timeout

I updated the plugin but that didn't change things

In the developer tools, I get errors like

```
Failed to get info about currently authenticated user! Error: net::ERR_CONNECTION_REFUSED

VM209 plugin:obsidian-imgur-plugin:511 Failed to get info about currently authenticated user! Error: net::ERR_CONNECTION_REFUSED

Failed imgur request:  Error: net::ERR_CONNECTION_REFUSED

TypeError: Cannot read properties of undefined (reading 'editor')
```

![2024-02-14 202402142054 - Juanchopedia - Obsidian v1 5 3](https://github.com/gavvvr/obsidian-imgur-plugin/assets/459464/9237caf0-083f-4b16-bc64-176cca186ba0)

**Comment by gavvvr** [comment]
Hi @jmfayard

Sorry to hear you experience issues with the plugin.

> Imgur ? The plugin ? I have uploaded too much ?

Definitely not the 1st two, I've just checked it and uploads work for me. Also, I did not ever hear you could "upload too much", but I can imagine it to be possible. Try switching to "anonymous upload" and see if it will work or not without your account. Also, try using VPN to change your IP. Also, I would suggest to try clean vault for debugging, without other plugins installed (who knows, maybe there is some conflict between them)
